client/views/components/generic/time-ago-in-words/TimeAgoInWords.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {PureComponent } from 'react';
+import { string, object  } from 'prop-types';
+import { BaseEntityQuery } from 'utils/genny';
+import distanceInWords from 'date-fns/distance_in_words';
+
+class TimeAgoInWords extends PureComponent {
+    static propTypes = {
+        code: string,
+        style: object
+    }
+    renderFromNow = (code) => {
+      if (code) {
+        let createdAt = BaseEntityQuery.getBaseEntity(code);
+        if(createdAt && createdAt.created) { 
+          const date = createdAt.created;
+          return distanceInWords(date, Date.now());
+        }
+        else { 
+          return null;
+        }
+    }
+    else { 
+      return null;
+    }
+  }
+
+    render() {
+      const {style, code} = this.props;
```

Comment:
change to use `root` and one of either `attribute` or `field`, along with `getBaseEntityAttribute` and `getBaseEntityField` respectively, so that the component can be used to display any date attribute, not just the creation date